tests/test_core.py

```diff
@@ -78,21 +78,55 @@ def test_load_image_with_unicode_filename():
 
 
 def test_load_image_with_numpy_array():
-    a = numpy.zeros((720, 1280), dtype=numpy.uint8)
+    a = numpy.zeros((720, 1280, 3), dtype=numpy.uint8)
     img = stbt.load_image(a)
     assert (a.__array_interface__["data"][0] ==
             img.__array_interface__["data"][0])
     assert img.filename is None
     assert img.relative_filename is None
     assert img.absolute_filename is None
 
+    a = numpy.zeros((720, 1280), dtype=numpy.uint8)
+    b = numpy.zeros((720, 1280, 1), dtype=numpy.uint8)
+    c = numpy.zeros((720, 1280, 3), dtype=numpy.uint8)
+    d = numpy.zeros((720, 1280, 4), dtype=numpy.uint8)
+
+    channels = [(1,), (3,), (4,), (1, 3), (1, 4), (3, 4), (1, 3, 4)]
+    for x in [a, b, c, d]:
+        for y in channels:
+            img = stbt.load_image(x, color_channels=y)
+            assert img.shape[2] in y
+            assert isinstance(img, stbt.Image)
```

Comment:
Nice test. I have pushed a similar test but passing a filename instead of a numpy array.